site/sfguides/src/build-and-deploy-snowpark-ml-models-using-streamlit-snowflake-notebooks/build-and-deploy-snowpark-ml-models-using-streamlit-snowflake-notebooks.md

```diff
@@ -12,116 +12,50 @@ tags: Data Science, Data Applications, Machine Learning, Streamlit, Snowpark, No
+* Alternatively, you can download the datasets from from UC Irvine Dataset Repository
   * [Abalone dataset](https://archive.ics.uci.edu/dataset/1/abalone) - This dataset will be used for the regression model (target - Rings)
 Abalone.data needs to be converted into a csv for it to work
   * [Diabetes dataset](https://archive.ics.uci.edu/dataset/891/cdc+diabetes+health+indicators) - This dataset will be used for the classification model (target - Diabetes_binary).
   * Please be sure to download the dataset with the name diabetes_binary_5050split_health_indicators_BRFSS2015.csv
 
 ### Creating Objects
 * Navigate to Worksheets, click "+" in the top-right corner to create a new Worksheet, and choose "SQL Worksheet".
-* Paste and run the following SQL in the worksheet to create Snowflake objects (warehouse, database, schema, raw tables).
-
-```sql
-USE ROLE sysadmin;
-
--- creating database and schema to for tables
--- creating warehouse for data ingestion
-
--- create database
-CREATE OR REPLACE DATABASE ML_SIDEKICK;
-
--- create schema
-CREATE OR REPLACE SCHEMA TEST_DATA;
-
--- create ml_sidekick_load_wh
-CREATE OR REPLACE WAREHOUSE ml_sidekick_load_wh
-	WAREHOUSE_SIZE = 'XSMALL'
-	WAREHOUSE_TYPE = 'standard'
-	AUTO_SUSPEND = 60
-	AUTO_RESUME = TRUE
-INITIALLY_SUSPENDED = TRUE
-COMMENT = 'ml-sidekick standard warehouse for data loading';
-
-USE WAREHOUSE ml_sidekick_load_wh;
-
--- create file format and tables 
--- Assuming the files are csv format if not please see this documentation here to create the correct file format https://docs.snowflake.com/en/sql-reference/sql/create-file-format#syntax
-
-
--- create abalone table
-CREATE OR REPLACE TABLE ML_SIDEKICK.TEST_DATA.ABALONE
-(
-SEX VARCHAR,
-LENGTH NUMBER,
-DIAMETER NUMBER,
-HEIGHT NUMBER,
-WHOLE_WEIGHT NUMBER,
-SHUCKED_WEIGHT NUMBER,
-VISCERA_WEIGHT NUMBER,
-SHELL_WEIGHT NUMBER,
-RINGS INTEGER
-);
-
--- create diabetes table
-CREATE OR REPLACE TABLE ML_SIDEKICK.TEST_DATA.DIABETES
-(
-Diabetes_binary INTEGER,
-HighBP INTEGER,
-HighChol INTEGER,
-CholCheck INTEGER,
-BMI INTEGER,
-Smoker INTEGER,
-Stroke INTEGER,
-HeartDiseaseorAttack INTEGER,
-PhysActivity INTEGER,
-Fruits INTEGER,
-Veggies INTEGER,
-HvyAlcoholConsump INTEGER,
-AnyHealthcare INTEGER,
-NoDocbcCost INTEGER,
-GenHlth INTEGER,
-MentHlth INTEGER,
-PhysHlth INTEGER,
-DiffWalk INTEGER,
-Sex INTEGER,
-Age INTEGER,
-Education INTEGER,
-Income INTEGER
-);
-```
+* Paste and run the SQL from (https://github.com/Snowflake-Labs/sfguide-build-and-deploy-snowpark-ml-models-using-streamlit-snowflake-notebooks/blob/main/deploy_sis_app.sql) in the worksheet to create Snowflake objects (Streamlit in Snowflake application, warehouse, database, schema, raw tables).
+
+You will likely need a role with necessary permissions to create the Git integration. If you do not have the necessary permissions, please reach out to your Snowflake administrator.
```

Comment:
Is it worth linking to the docs markdown where it states the necessary roles? E.g. for integration OR a doc explicitly for end-to-end git integration?

Reply:
This is done in https://github.com/Snowflake-Labs/sfquickstarts/pull/1810/commits/01919034008b3d27e6de5d0ad8a34d6c9ad84a1e, thank you!